app/models/user.rb

```diff
@@ -3,4 +3,20 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  has_many :posts
+  has_many :likes
+  has_many :comments
+
+  has_many :friend_requests, dependent: :destroy
+  has_many :pending_friends, through: :friend_requests, source: :requested
+
+  has_many :friendships, dependent: :destroy
+  has_many :friends, through: :friendships, class_name: 'User'
+
+  validates :name, presence: true, length: { maximum: 50 }
```

Comment:
I tried to create a user with the attributes as you specified in your validations:

![image](https://user-images.githubusercontent.com/6015315/57527555-9082ef00-7362-11e9-85a7-9a12ee3e3304.png)

But as you see I got an error. It appears you need to work on your user model still. 

It appears you are putting an attribute in your user model that you didn't specify in your schema. Please fix this.